test/FastTests/NewClient/Query.cs

```diff
@@ -55,81 +55,6 @@ public void Query_With_Where_Clause()
                 }
             }
         }
-
-        [Fact]
-        public void Query_By_Index()
```

Comment:
It was failing because SaveChanges() expected an Id field in the entity, which will be fixed in Efrat's PR.